### DISCUSSION TITLE
SMS Provider Configuration

### DISCUSSION THREAD

**Discussion by StephenFeddes** [YetiForceCompany/YetiForceCRM#17073] [How to]
My company wants me to implement functionality that will allow SMS text messages to be sent to and from YetiForce. I saw some documentation online, but the page no longer exists. Could you provide some documentation on how I can get set up with that?

**Comment by mariuszkrzaczkowski** [comment]
little details on how the feedback to the CRM should look like.
What you want is no problem, it just depends on the operator/SMS provider, currently there is only SMSAPI (https://www.smsapi.com/en) Polish provider in the system.
There is no problem to add more, only someone has to write a connector

https://gitdeveloper.yetiforce.com/index.php?module=SMSNotifier&parent=Settings&view=List

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10198654/cd545ae6-ef8b-4559-87d4-b24b994460d5)
